### DISCUSSION TITLE
Welcome to autosar Discussions!

### DISCUSSION THREAD

**Discussion by cogu** [cogu/autosar#107] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by qg020** [comment]
Thank you for adding a discussions page!  I've got questions.  :-)

I am a developer at Cummins, Inc using this package to amend arxml in one project with AUTOSAR elements from another project (or brand new) using Excel spreadsheets as an intermediary.  The script is working with version 0.4 and I'm updating it with 0.5.x as it is released.  It looks like I'll get really close to what I need with 0.5.3.
